platform/viewer/src/routes/WorkList/WorkList.tsx

```diff
@@ -324,13 +324,13 @@ function WorkList({
           seriesTableDataSource={
             seriesInStudiesMap.has(studyInstanceUid)
               ? seriesInStudiesMap.get(studyInstanceUid).map(s => {
-                  return {
-                    description: s.description || '(empty)',
-                    seriesNumber: s.seriesNumber ?? '',
-                    modality: s.modality || '',
-                    instances: s.numSeriesInstances || '',
-                  };
-                })
+                return {
```

Comment:
@sedghi - this type of formatting change is being introduced by the eslint stage.  Either we should disable it or we should enforce it.  I don't care either way, but just FYI